.gitignore

```diff
@@ -43,9 +43,20 @@ local_test/*
 !local_test/index.html
 local_test/js/*
 local_test/css/*
+# folder of projects to test 
+local_test/intesting
```

Comment:
@tbushman I don't know where that file is, but I'm happy to change update it if someone can point me in the right direction.